src/set_theory/game/pgame.lean

```diff
@@ -1149,9 +1461,20 @@ using_well_founded { dec_tac := pgame_wf_tac }
 
 instance : has_sub pgame := ⟨λ x y, x + -y⟩
 
-@[simp] theorem sub_zero (x : pgame) : x - 0 = x + 0 :=
+@[simp] theorem sub_zero_eq_add_zero (x : pgame) : x - 0 = x + 0 :=
 show x + -0 = x + 0, by rw neg_zero
 
+lemma sub_zero (x : pgame) : x - 0 ≡ x :=
```

Comment:
```suggestion
lemma sub_zero (x : pgame) : x + 0 ≡ x :=
```

Reply:
There is already a lemma `pgame.add_zero`. This lemma is just for convenience.